stan-dev/stan
# Bugfix/textbar

#### Submisison Checklist

- [ ] Run unit tests: `./runTests.py src/test/unit` Does not touch C++
- [ ] Run cpplint: `make cpplint` Does not touch C++
- [ X ] Declare copyright holder and open-source license: see below

#### Summary

The Stan Manual index has `\textbar` constructions, which mess up ASCII representations of the index.

#### Intended Effect

Have `\textbar` appear as `|` in `rstan::lookup("normal")`

#### How to Verify

```
make clean-all
make manual
grep "textbar" doc/stan-functions-*.txt
```
should not match anything.

#### Side Effects

None

#### Documentation

None

#### Reviewer Suggestions

Anyone who knows what 
```
sed 's/\\textbar \\/|/g'
```
does.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Trustees of Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
